scripts/generate.github.auth.ts

```diff
@@ -0,0 +1,52 @@
+import path from 'path';
+import fs from 'fs-extra';
+
+import { stripSpaces } from 'common/lib/utilities/string';
+import * as SimplePrompt from './helpers/simple.prompt';
+
+(async () => {
+  console.log(
+    stripSpaces(`
+        This script will set you up with doing local GitHub auth.
+        
+        If you haven't already, go to https://github.com/settings/developers
+        and click "Register new application".
+        Give it a name like "ScriptLab Local Dev",
+        with a Homepage and Auth callback URL of https://localhost:3000
+        
+        Once you've done that, please enter the following details from your app registration:
+    `),
+  );
+
+  let clientId = await SimplePrompt.promptCustomText('Client ID:', { required: true });
```

Comment:
Seems like a bit of overkill..

Reply:
Why?  Have you tried running it.  Super handy...

Reply:
Because as a dev, I'm not afraid to edit a text file... xD